lib/controllers/root-controller.js

```diff
@@ -488,8 +497,30 @@ export default class RootController extends React.Component {
 
   async viewChangesForCurrentFile(stagingStatus) {
     const editor = this.props.workspace.getActiveTextEditor();
-    const absFilePath = editor.getPath();
+    const absFilePath = await realPath(editor.getPath());
     const repoPath = this.props.repository.getWorkingDirectoryPath();
+    if (repoPath === null) {
+      const [projectPath] = this.props.project.relativizePath(editor.getPath());
+      const notification = this.props.notificationManager.addInfo(
+        "Hmm, we don't have anything to compare this to.",
+        {
+          description: 'Create a Git repository in order to view changes to the files in your project.',
```

Comment:
I wonder if there's more we can do that make it clear that we're presenting this to the user as an _option_, but that they don't have to create a repository if they don't want to. How about something like this?

> You can create a Git repository to track changes to the files in your project.